src/legacy/ui/public/agg_types/controls/order_and_size.html

```diff
@@ -1,36 +0,0 @@
-<div>
-  <div class="visEditorAgg__formRow--flex">
-    <div ng-if="agg.type.params.byName.order && aggParam.options" class="form-group">
```

Comment:
Do we handle this condition?

Reply:
This check was here because of the template usage in `significant terms` agg also.
There is no necessity to check it now